src/infra/marketplace/marketplaceCmdRepo.go

```diff
@@ -508,23 +552,40 @@ func (repo *MarketplaceCmdRepo) UninstallItem(
 		return err
 	}
 
-	if deleteDto.ShouldUninstallServices {
-		err = repo.uninstallUnusedServices(installedItem.Services)
+	catalogItem, err := repo.marketplaceQueryRepo.ReadCatalogItemBySlug(
+		installedItem.CatalogSlug,
+	)
+	if err != nil {
+		return err
+	}
+
+	if len(catalogItem.FilesToDelete) > 0 {
```

Comment:
Why? Why would you call parseUninstall only if filesToDelete is > 0. That makes no sense at all. Again, coding for it to work and not for people to make sense of what and why you're doing.

Reply:
It was removed along with the previous code.